tests/efi-vars-editor-vm

```diff
@@ -0,0 +1,73 @@
+#!/bin/sh
+set -eux
+
+architecture="$(uname -m)"
+if [ "${architecture}" != "x86_64" ] && [ "${architecture}" != "aarch64" ]; then
+  echo "Skipping test as EFI is not supported on ${architecture}"
+fi
+
+# Install LXD
+install_lxd
+
+# Configure LXD
+lxc network create lxdbr0
+lxc profile device add default eth0 nic network=lxdbr0
+
+poolName="vmpool$$"
+poolDriver=dir
+
+echo "==> Create storage pool using driver ${poolDriver}"
+lxc storage create "${poolName}" "${poolDriver}"
+
+echo "==> Create VM and boot"
+lxc launch "${TEST_IMG:-ubuntu-daily:22.04}" v1 --vm -s "${poolName}"
```

Comment:
Pleas use `ubuntu-minimal-daily:22.04" here, it's much faster on slow CI.

Reply:
Sure. Thanks!